vendor/modules.txt

```diff
@@ -1976,6 +1976,7 @@ k8s.io/component-helpers/storage/volume
 ## explicit
 k8s.io/controller-manager/app
 k8s.io/controller-manager/config
+k8s.io/controller-manager/config/v1
 k8s.io/controller-manager/config/v1alpha1
```

Comment:
Would be good to clean up the v1alpha1 API. No deprecation needed according to policy. Lessens the API surface we need to maintain. Does not need to happen in this PR.

Reply:
Good catch, we should open an issue to track

Reply:
Or maybe just remove it here. https://kubernetes.io/docs/reference/using-api/deprecation-policy/ `v1alpha` can be removed at will.